packages/server/src/hub/handlers/handle-ongoing-process-action.ts

```diff
@@ -132,12 +138,19 @@ async function handleCommitmentsReceived(action: CommitmentsReceived) {
       ledgerCommitmentRound,
       currentCommitment && asConsensusCommitment(currentCommitment),
     );
-    return communication.sendCommitmentsReceived(
-      nextParticipant,
-      processId,
-      [...incomingCommitments, { commitment, signature: (signature as unknown) as string }],
-      action.protocolLocator,
-    );
+    const response = [];
+    const otherParticipants = participants.filter(p => participants.indexOf(p) !== ourIndex);
+    for (const participant of otherParticipants) {
```

Comment:
This is a work around to the #651. 

Having the hub broadcast messages to all participants is a simple fix but it ties us to a model where the hub can message every participant (IE: not a REST endpoint)